java/src/main/java/me/everything/inbloom/BloomFilter.java

```diff
@@ -24,11 +24,12 @@
     // These fields are part of the public interface of this structure.
     // Client code may read these values if desired. Client code MUST NOT
     // modify any of these.
-    int entries;
-    double error;
-    int bits;
-    int bytes;
-    int hashes;
+    public int entries; 
```

Comment:
@bergundy they were package private before this PR. If the test is running in the same package, I suggest we keep them package-private, and final is just a nice addition.